policyengine_us/variables/gov/states/ar/tax/income/deductions/itemized/ar_post_secondary_education_tuition_deduction.py

```diff
@@ -0,0 +1,45 @@
+from policyengine_us.model_api import *
+
+
+class ar_post_secondary_education_tuition_deduction(Variable):
```

Comment:
make a `ar_post_secondary_education_tuition_deduction_person` (person level) until line 43 and just use an adds function for `ar_post_secondary_education_tuition_deduction`